api/v1alpha1/release_types.go

```diff
@@ -138,128 +114,128 @@ type Release struct {
 	Status ReleaseStatus `json:"status,omitempty"`
 }
 
-// HasStarted checks whether the Release has a valid start time set in its status.
-func (r *Release) HasStarted() bool {
-	return r.Status.StartTime != nil && !r.Status.StartTime.IsZero()
+// HasDeploymentFinished checks whether the Release has been successfully deployed via GitOps.
+func (r *Release) HasDeploymentFinished() bool {
+	condition := meta.FindStatusCondition(r.Status.Conditions, deployedConditionType.String())
+	return condition != nil && condition.Status != metav1.ConditionUnknown
```

Comment:
this reintroduces bugs I already fixed. The logic for deployment status needs to be left as is, reflecting how the gitops code currently works, not changed to what a ticket suggests it should be

Reply:
You sure about this? Take into account this is referring to the new condition, not the GitOps one. For the new condition, True and False are final conditions which means that the deployment has finished (successfully or not).